arch/arm/core/aarch32/irq_manage.c

```diff
@@ -18,9 +18,8 @@
 #include <arch/cpu.h>
 #if defined(CONFIG_CPU_CORTEX_M)
 #include <arch/arm/aarch32/cortex_m/cmsis.h>
-#elif defined(CONFIG_CPU_CORTEX_R)
-#include <device.h>
-#include <irq_nextlevel.h>
+#elif defined(CONFIG_CPU_CORTEX_A) || defined(CONFIG_CPU_CORTEX_R)
```

Comment:
Is CORTEX_A here necessary?

Reply:
Since we do not have AArch32 Cortex-A support yet, it is, strictly speaking, not necessary; but there is already a branch with preliminary AArch32 Cortex-A support implemented and I suppose adding this check a priori would not be a bad thing.